ttnn/cpp/ttnn/operations/pool/maxpool/device/kernels/compute/max_pool_multi_core.cpp

```diff
@@ -40,47 +31,45 @@
         UNPACK(( DPRINT << "++++++" << ENDL() ));
     }
 
-    // inline void print_cb_details(uint32_t cb_id) {
-    //     DPRINT << "cb_id " << cb_id << ": { "
-    //             << "size: " << cb_interface[cb_id].fifo_size << ", "
-    //             << "limit: " << cb_interface[cb_id].fifo_limit << ", "
-    //             << "page_size: " << cb_interface[cb_id].fifo_page_size << ", "
-    //             << "num_pages: " << cb_interface[cb_id].fifo_num_pages << ", "
-    //             << "rd_ptr: " << cb_interface[cb_id].fifo_rd_ptr << ", "
-    //             << "wr_ptr: " << cb_interface[cb_id].fifo_wr_ptr << ", "
-    //             << "wr_tile_ptr: " << cb_interface[cb_id].fifo_wr_tile_ptr << " }" << ENDL();
-    // }
+    inline void print_cb_details(uint32_t cb_id) {
+        DPRINT << "cb_id " << cb_id << ": { "
+                << "size: " << cb_interface[cb_id].fifo_size << ", "
+                << "limit: " << cb_interface[cb_id].fifo_limit << ", "
+                << "page_size: " << cb_interface[cb_id].fifo_page_size << ", "
+                << "num_pages: " << cb_interface[cb_id].fifo_num_pages << ", "
+                << "rd_ptr: " << cb_interface[cb_id].fifo_rd_ptr << ", "
+                << "wr_ptr: " << cb_interface[cb_id].fifo_wr_ptr << ", "
+                << "wr_tile_ptr: " << cb_interface[cb_id].fifo_wr_tile_ptr << " }" << ENDL();
+    }
 #endif
 
-template<uint32_t in_ntiles_hw, uint32_t in_ntiles_c, uint32_t out_ntiles_c, uint32_t nblocks, bool is_partial_tile, uint32_t split_reader, uint32_t unpA_face_r_dim>
+template<uint32_t in_ntiles_hw, uint32_t in_ntiles_c, uint32_t out_ntiles_c, bool is_partial_tile, uint32_t split_reader, uint32_t unpA_face_r_dim, uint32_t in_nblocks_c>
 inline void reduce_h_fused(
```

Comment:
"fused" in reduce_h_fused means tilize/untilize on the fly right?

Reply:
yes